Sources/SWBBuildService/BuildDependencyInfo.swift

```diff
@@ -434,9 +435,11 @@ extension BuildDependencyInfo {
     /// - remark: This is written somewhat generically (with the callback blocks) in the hopes that `LinkageDependencyResolver.dependencies(for:...)` can someday adopt it, as the general approach was stolen from there.
     package static func findLinkedInputsFromBuildSettings(_ settings: Settings, addFramework: @Sendable (TargetDependencyInfo.Input) async -> Void, addLibrary: @Sendable (TargetDependencyInfo.Input) async -> Void, addError: @Sendable (String) async -> Void) async {
         await LdLinkerSpec.processLinkerSettingsForLibraryOptions(settings: settings) { macro, flag, stem in
-            await addFramework(TargetDependencyInfo.Input(inputType: .framework, name: .stem(stem), linkType: .searchPath, libraryType: .dynamic))
+            let libType: TargetDependencyInfo.Input.LibraryType = (flag == "upward_framework") ? .upward : .dynamic
```

Comment:
Is `upward_framework` missing a leading `-` here?

Reply:
Yes, that was the problem. (Also that I didn't have the right SDKs installed locally so that I didn't see the failure.)